packages/prettier-plugin-java/src/printers/classes.ts

```diff
@@ -652,15 +658,16 @@ export class ClassesPrettierVisitor extends BaseCstPrettierPrinter {
     return printTokenWithComments(this.getSingle(ctx) as IToken);
   }
 
-  throws(ctx: ThrowsCtx) {
+  throws(ctx: ThrowsCtx, opts?: { throwsGroupId?: symbol }) {
```

Comment:
Should throwsGroupId really be optional, or should we force it to be defined so we are "forced" to align throws formating for constructor/interface methods, etc. as well ?

Reply:
Good point! I totally forgot to implement this for interface methods. I went ahead and made this required as you suggested, and implemented the same throws formatting for interface methods.